frontend/src/store/task.store.ts

```diff
@@ -0,0 +1,25 @@
+import { create } from 'zustand';
+import { Task } from '../pages/projectView/ProjectView';
+
+type TaskStoreType = {
+  task: Task;
+  setTask: (task: Task) => void;
+};
+
+export const initialValue = {
+  id: '',
+  description: '',
+  status: '',
+  due_date: '',
+  assignee: '',
+  completed: false,
+  priority: '',
+  project_id: 0,
+  user_id: '',
+};
+export const useTaskStore = create<TaskStoreType>()((set) => ({
+  task: initialValue,
+  setTask: (task: Task) => set({ task }),
+}));
+
+export default useTaskStore;
```

Comment:
what is the purpose of this code? I don't see it used anywhere

Reply:
I'm using this store in order to manage the task state. In this way 'task' and 'setTask' will be available throughout the entire project.